pandas/tests/tools/test_to_datetime.py

```diff
@@ -1715,6 +1783,23 @@ def test_to_datetime_fixed_offset(self):
         result = to_datetime(dates)
         assert result.tz == fixed_off
 
+    @pytest.mark.parametrize(
+        "date",
+        [
+            ["2020-10-26 00:00:00+06:00", "2020-10-26 00:00:00+01:00"],
+            ["2020-10-26 00:00:00+06:00", Timestamp("2018-01-01", tz="US/Pacific")],
+            [
+                "2020-10-26 00:00:00+06:00",
+                datetime(2020, 1, 1, 18, tzinfo=pytz.timezone("Australia/Melbourne")),
+            ],
+        ],
+    )
+    def test_to_datetime_mixed_offsets_with_utc_false_deprecated(self, date):
```

Comment:
nice, great to test these different combinations

Reply:
thanks